coco_2_python_basics.ipynb

```diff
@@ -3,45 +3,57 @@
   {
```

Comment:
<p>Before doing your homework, <em>check</em> that you have a _remote  repository_ in your GitHub account called <code>coco_coding_homework</code>. </p><p>If you don't have it, _create this remote repository_ according to what you learned in the  <em>_git </em>lesson_. </p><p>In this repository you will push all the notebooks with the solution of the proposed activities.</p><p> </p><p>Then, create a notebook called <code>homework_week_3.ipynb</code> into the _local repository_ related with the remote repository. </p><p>There you will solve the activities proposed below and <em>push </em>it to the remote repository in GitHub. </p><p>In this way, one of the trainers will be able to see your answers and give you feedback. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CodeToCommunicate/CoCoLessons/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='127'/>